src/gui/guiFormSpecMenu.cpp

```diff
@@ -3208,8 +3208,8 @@ void GUIFormSpecMenu::regenerateGui(v2u32 screensize)
 			offset = v2s32(0,0);
 		}
 
-		double gui_scaling = g_settings->getFloat("gui_scaling");
-		double screen_dpi = RenderingEngine::getDisplayDensity() * 96;
+		const double gui_scaling = rangelim(g_settings->getFloat("gui_scaling"), 0.5f, 42.0f);
```

Comment:
this comes up a lot, maybe add a helper in some gui header that computes this

Reply:
:heavy_check_mark: